### PR TITLE
expand on the usage documentation to better support all the parameters

### PR DESCRIPTION
**Pull request by rdhariwal**
Before:
image

After:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
-Creates the following resources:
+## Prerequisites
+* Ensure [terraform](https://www.terraform.io/intro/getting-started/install.html) is installed
+* Ensure domain is registered in [route53](https://aws.amazon.com/route53/)
+* Ensure an s3 bucket exists and SES has write permissions to it
+* If you have an existing rule set you can skip creating the dependent resource
+* route53 zone id can be obtained by looking up the domain in route53 service
+
+## Getting Started
+1. Import the module called `ses_domain` and update its source property to `github.com/trussworks/terraform-aws-ses-domain` and run `terrafrom init`
```
Comment (rdhariwal) [inline README.md]: should use the hashicorp module and not github path?!?

Comment (brainsik) [reply README.md]: Yes. Use the registry.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,47 @@
...
+* SES out of the box locks the service in development mode; please see this documentation on how to make it production ready. Until the service is in production mode you can only send emails to confirmed email accounts denoted in `from_addresses`
 
 ## Usage
 
 ```hcl
 module "ses_domain" {
   source = "../../modules/aws-ses-domain"
+  domain_name        = "example.com"
+  mail_from_domain   = "email.example.com"
+  route53_zone_id    = "Z123456789"
```
Comment (brainsik) [inline README.md]: I think I'd include the data source lookup above and reference it here. Up to you.

Comment (rdhariwal) [reply README.md]: > For the zone id?

yep makes sense that's what we ended up doing in our infra code

File: README.md (hunk 1 above)
Comment (rdhariwal) [reply README.md]: > So, I can’t tell if this instruction is correct or not?

It is correct as in the github link will work, I think most folks prefer published packages though. So I ll update this section 👍

**Comment by dynamike** [comment]
`1.0.1` is fine with me.

**Pull request merged**